dbscripts/xml/upgrade/2.4.6_update.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE data SYSTEM "../../../lib/pkp/dtd/xmlData.dtd">
+
+<!--
+  * 2.4.6_update.xml
+  *
+  * Copyright (c) 2013-2015 Simon Fraser University Library
+  * Copyright (c) 2003-2015 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * 2.4.6 database updates XML file.
+  -->
+<data>
+	<!-- Increase length of password column for newer hashing algorithms -->
+	<sql>
+		<query>ALTER TABLE users MODIFY COLUMN password varchar(255)</query>
```

Comment:
The upgrade script syncs the XML schema descriptors against the database, and should automatically alter tables if a characteristic like the column length needs to change.